### PR TITLE
Release/2.4.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added
- Public: Added `documentTypes` to the `document` step options, which allows to filter the document types.

### Changed
- Internal: Refactored layout to better handle presence of header and footer elements.
- Internal: On cross device client clear error message when configuration is received.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-337-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-337-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "2.3.0",
+  "version": "2.4.0-rc.1",
```
Comment (dawiddylowicz) [inline package.json]: needs to be changed to 2.4.0 before release

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,16 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [2.3.0] - 17-04-2018
+## [2.4.0]
```
Comment (dawiddylowicz) [inline CHANGELOG.md]: Date needs to be added before release

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-337-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**